iot-hub/Samples/device/ImportExportDevicesSample/ProcessImpExpCommands.cs

```diff
@@ -0,0 +1,353 @@
+﻿using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Common;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Blob;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ImportExportDevices
+{
+    public static class IoTHubDevices
+    {
+
```

Comment:
Nit: formatting - please follow the coding style from https://github.com/Azure/azure-iot-sdk-csharp/blob/master/doc/coding-style.md.